src/ds-style-fix.scss

```diff
@@ -686,8 +686,8 @@ div.modal form {
   text-decoration: none;
 }
 
-.footer-list > li {
-  margin: 0px !important;
+.footer-list li {
```

Comment:
Might as well move this to the footer scss

Reply:
@garyluu Tried to do that but unfortunately the check yells at me for using `!important `in component style sheet